evalml/model_understanding/graphs.py

```diff
@@ -599,6 +616,33 @@ def partial_dependence(pipeline, X, features, percentiles=(0.05, 0.95), grid_res
     return data
 
 
+def _update_fig_with_two_way_partial_dependence(_go, fig, label_df, part_dep, features, is_categorical,
+                                                label=None, row=None, col=None):
+    y = label_df.index
```

Comment:
Before we were setting y as the columns and x as the index but that's backwards. The row index is the "vertical" direction so it corresponds to the y axis. I think this is what was causing the NaNs in our docs mentioned in #2115. After the switch, they are gone:

![image](https://user-images.githubusercontent.com/41651716/114116608-fc559580-98b2-11eb-895b-5b4a81ac02fc.png)

Reply:
lol i git blame myself..I remember feeling some qualms with this and should have done a better job checking.  Thanks for fixing it for me.